coffeelint.json

```diff
@@ -37,7 +37,7 @@
         "value": "unix"
     },
     "max_line_length": {
-        "value": 80,
+        "value": 100,
```

Comment:
I'd rather leave this at 80, but it's not a big deal to prevent merging - I can fix it after master's settled down.